### PR TITLE
Move to Gradle [WIP]

### PR DESCRIPTION
**Pull request by DenWav**

### CODE REVIEW

File: sponge/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+apply {
+    plugin("com.github.johnrengelman.shadow")
+}
+
+repositories {
+    mavenCentral()
+    maven {
+        setUrl("http://repo.spongepowered.org/maven")
```
Comment (kashike) [inline sponge/build.gradle.kts:40]: make it https while you're at it

**Comment by ghost** [comment]
Why?

**Comment by DenWav** [comment]
Forge support requires Gradle.

**Comment by DarkSeraphim** [comment]
@DemonWav that doesn't mean a library has to switch to gradle, they're (on build level, as in, how they're built), quite independent from each other.

**Comment by JoachimVandersmissen** [comment]
https://i.imgur.com/5dPV05h.gif

**Comment by aikar** [comment]
just gonna stick on maven for base, and can see about gradle for forge specifically if it even happens

**Comment by DenWav** [comment]
I was never going to finish this anyways lol.

**Pull request closed without merging**